esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
Also, this file should either be called `default_email.html`, if it's html, `default_email.txt`, if it's text, or `default_email` if it could be either.